airbyte-integrations/connectors/source-planhat/pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[build-system]
+requires = [ "poetry-core>=1.0.0",]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+version = "0.1.0"
+name = "source-planhat"
+description = "Source implementation for Planhat."
+authors = [ "Airbyte <contact@airbyte.io>",]
```

Comment:
suggestion: I thought we are the authors. You should probably update this to the JAAS mailing list and the other URLs below.